### ISSUE TITLE
[Feat] Auras

### ISSUE THREAD

**Issue by NekroDarkmoon** [Pjb518/FoundryVTT-Level-Up-Official#199]

**Comment by Pjb518** [comment]
This feature with largely emulate the [Active Auras](https://foundryvtt.com/packages/ActiveAuras) module. The goal here is to provide support for things like herald/paladin auras and spells like Pass without Trace, which provide some benefit / impediment to creatures within some particular radius of an actor.

Important features we need to consider are:

- A toggle to determine whether the actor emitting the aura benefits from the aura.
- An option to determine whether this hits allies, enemies, or potentially both.
- Elevation considerations (possibly a feature that comes after the initial implementation).
- An option to determine whether the aura is blocked by obstacles (likely also a feature that comes after the initial implementation).

**Comment by JPMeehan** [comment]
This might get scooped by v12 if Advanced Measure Templates wins the vote.

**Comment by NekroDarkmoon** [comment]
The actual application of effects in the aura will still need to be handled by the system. Plus the actual v12 release is months away xD

**Comment by Pjb518** [comment]
Yeah, these are rather different features. The Advanced Measure Templates is just about having templates that follow your tokens around, not having things within that area be affected by an Active Effect 🙂

We may well be able to piggyback off of this feature for targeting purposes though.

**Comment by NekroDarkmoon** [comment]
Blocked by #260 .
